codespell_lib/data/dictionary.txt

```diff
@@ -57862,6 +57862,7 @@ verions->versions
 veriosn->version
 veriosns->versions
 verious->various
+verision->version,derision
```

Comment:
Perhaps add:
```
verisions->versions, derisions,
```
or:
```
verisions->versions
```

Reply:
Could it be a misspelling of `revision` too?

Reply:
ah, missed the syntax... 

> Could it be a misspelling of `revision` too?

ha -- great catch -- just swapping of 2 letters, I didn't immediately see it even! nice. I even see it possibly happening more likely than for some rare `derision` so will place it 2nd

ok, let's do/accept suggestion

```suggestion
verision->version, revision, derision,
```

Reply:
grr... github is giving me hard time... need to run, will try to merge again later:

![image](https://github.com/codespell-project/codespell/assets/39889/8aa35ca5-8c5a-4f29-ba06-2f919591a68b)